src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -180,6 +170,7 @@ object DepositHandler extends BagValidationExtension {
       _ <- checkBagVirtualValidity(bagDir)
       props <- DepositProperties(id)
       _ <- props.setState(SUBMITTED, "Deposit is valid and ready for post-submission processing")
+      _ <- props.setBagName(File(bagDir.getPath))
```

Comment:
Just keep it easy: converting to better-files isn't really necessary here. `java.io.File` has something to get the name of a file as well.

Reply:
ok